@kiva/kv-components/vue/KvCommentsContainer.vue

```diff
@@ -1,19 +1,30 @@
 <template>
-	<kv-comments-add
-		:user-image-url="userImageUrl"
-		:user-display-name="userDisplayName"
-		@add-comment="comment"
-	/>
+	<div>
+		<kv-comments-add
+			:user-image-url="userImageUrl"
+			:user-display-name="userDisplayName"
+			@add-comment="comment"
+		/>
+		<kv-comments-list
+			v-if="comments"
+			:comments="comments"
+			@[REPLY_COMMENT_EVENT]="comment"
+			@[LIKE_COMMENT_EVENT]="comment"
```

Comment:
These events likely need `.stop` for both the add and list components (I noticed duplicate events emitted when testing add).